apps/redi-talent-pool/src/pages/app/cv/CvListItemMoreOptionsMenu.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+
+import { Content } from 'react-bulma-components'
+import { Button, Popover } from '@material-ui/core'
+import { MoreHoriz as MoreHorizIcon } from '@material-ui/icons'
+
+interface CvListItemMoreOptionsMenuProps {
+  handleRenameClick(): void
+  handleDeleteClick(): void
+  handleDuplicateClick(): void
+}
+
+const CvListItemMoreOptionsMenu = (props: CvListItemMoreOptionsMenuProps) => {
```

Comment:
Really nice component :)

What do you think about preferring `function ComponentName(){}` over `const ComponentName = () => {}`? I tended to use the latter because of yey-let's-do-arrow-functions-everywhere but now I tend to the former because of let's-make-things-declarative-hence-make-components-more-distinct-by-using-the-function-keyword.

Reply:
Sounds good. I never had a strong opinion on either of these. I like the new syntax but I'm also aware of the advantages of the good old function syntax. I'm more in favor of consistency here so let's do that!